cfgov/jinja2/v1/ask-cfpb/category-page.html

```diff
@@ -29,7 +29,7 @@
                     block__flush-top
                     block__sub">
         <h1>
-            {{ category_icon.render(page.ask_category.name, '', 'is_round') }}
+            {{ category_icon.render(page.ask_category.name, 'is_round') }}
```

Comment:
`'is_round'` be `true`? I think it's okay as is and more descriptive using a string, but just checking.

Reply:
Yep, another good catch, sorry I didn't notice when removing the unused argument.